PANCHANGE_descriptives2020-07-14.Rmd

```diff
@@ -0,0 +1,226 @@
+---
+title: "PANCHANGE_data_preprocessing"
+author: "Katie Thompson, Topher Huebel, Molly Davies, Kirstin Purves"
+date: "14/07/2020"
```

Comment:
I wonder if we dont want to have this as the system date so that it refreshes each time we run the script and always gives the latest date we worked on it? What do you think @mollyrdavies @topherhuebel @knthompson26 ?

Reply:
Actually, in this instance, I'm going to make this a new issue.  We should discuss and possibly have a piece of work to resolve this.